Kernel/Bus/USB/USBConfiguration.cpp

```diff
@@ -14,6 +14,27 @@
 
 namespace Kernel::USB {
 
+USBConfiguration::USBConfiguration(USBConfiguration const& other)
+    : m_device(other.m_device)
+    , m_descriptor(other.m_descriptor)
+    , m_descriptor_index(other.m_descriptor_index)
+    , m_interfaces(other.m_interfaces)
+{
+    // FIXME: This can definitely OOM
```

Comment:
what would be the fix here, even? should we somehow hardcap the amount of USB devices?

(i don't see this as a blocker, but I would like to look into fixing it some day - i don't want serenity to be pwnable like a PS3 :p)

Reply:
there are two solutions:
1. Avoid this all together
2. Make a fallible